src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -126,11 +126,21 @@ MigrationState OutgoingMigration::GetState() const {
   return state_;
 }
 
+void OutgoingMigration::ClearSlotRanges() {
+  shard_set->pool()->AwaitFiberOnAll([this](util::ProactorBase* pb) {
+    if (auto* shard = EngineShard::tlocal(); shard) {
+      shard->db_slice().FlushSlots(GetSlots());
```

Comment:
why do we do flushslot for outgoing migration, but not for incoming?

Reply:
that's just me being incompetent :( please ignore, thanks

Reply:
@chakaz you are so far away from `incompetent`....